load.php

```diff
@@ -59,6 +59,9 @@
 	define( 'WP_CACHE_KEY_SALT', 'codecept' );
 	defined( 'WP_TESTS_DOMAIN' ) || define( 'WP_TESTS_DOMAIN', $_SERVER['HTTP_HOST'] ?? $default_host );
 
+	// Prevent tracking data to segment.
+	defined( 'CI' ) || define( 'CI', true );
```

Comment:
Does this mean that it's not possible to run tests locally with a different configuration than in CI?

For example, if I specify `extras.altis.environments.ci.modules.cloud.cavalcade: false`, will this be for CI only, or for local test runs as well?

Reply:
This is mainly geared at not tracking data to segment in test runs, locally you would have to do `CI=true composer dev-tools codecept run` for example to mimick CI environments

Reply:
To clarify this constant will not impact on the Altis config itself.

Reply:
Oh, I see, right.